### PR TITLE
zh-cn: Format /web/css using Prettier (part 3)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the `/web/css` folder of the Simplified Chinese locale using Prettier.  This is the third part.

### CODE REVIEW

File: files/zh-cn/web/css/margin-top/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ slug: Web/CSS/margin-top
 
 {{EmbedInteractiveExample("pages/css/margin-top.html")}}
 
-此属性对于*不*[*可替换*](/zh-CN/docs/Web/CSS/Replaced_element)的内联元素没有效果，比如 {{HTMLElement("span")}} 或 {{HTMLElement("code")}}。
+此属性对于_不_ [_可替换_](/zh-CN/docs/Web/CSS/Replaced_element) 的内联元素没有效果，比如 {{HTMLElement("span")}} 或 {{HTMLElement("code")}}。
```
Comment (yin1999) [inline files/zh-cn/web/css/margin-top/index.md:12]: We don't add space between Chinese words. Does prettier "like" this format?

```suggestion
此属性对于*不*[_可替换_](/zh-CN/docs/Web/CSS/Replaced_element)的内联元素没有效果，比如 {{HTMLElement("span")}} 或 {{HTMLElement("code")}}。
```

File: files/zh-cn/web/css/gradient/radial-gradient/index.md [hunk 2]
```diff
@@ -25,17 +25,18 @@ radial-gradient(circle at center, red 0, blue, green 100%)
 - `<ending-shape>`
   - : 渐变结束时的形状。圆形（渐变的形状是一个半径不变的正圆）或椭圆形（轴对称椭圆）。默认值为椭圆。
 - `<size>`
+
   - : 确定渐变结束形状的大小。如果省略，则默认为最远角。它可以显式给出，也可以通过关键字给出。出于关键字定义的目的，将梯度框边缘视为在两个方向上无限延伸，而不是有限线段。
 
-     对于它们的 `<size>`，圆形和椭圆渐变都接受以下关键字：
-     | 关键字              | 描述                                                                                                             |
+    对于它们的 `<size>`，圆形和椭圆渐变都接受以下关键字：
+    | 关键字 | 描述 |
```
Comment (yin1999) [inline files/zh-cn/web/css/gradient/radial-gradient/index.md:32]: markdown table here :)

File: files/zh-cn/web/css/css_selectors/using_the__colon_target_pseudo-class_in_selectors/index.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ original_slug: Web/Guide/CSS/Using_the_:target_selector
 
 {{CSSRef}}
 
-为了辅助标识那些指向文档特定部分链接的目标，[CSS3 选择器](http://www.w3.org/TR/css3-selectors/#target-pseudo) 引入了` {{ Cssxref(":target") }} `[伪类](/zh-CN/CSS/Pseudo-classes). Netscape 7.1 已经在 Netscape 系列中加入了这个伪类的支持，这一新的举措让页面作者能够辅助用户在较大的页面中定位。
+为了辅助标识那些指向文档特定部分链接的目标，[CSS3 选择器](http://www.w3.org/TR/css3-selectors/#target-pseudo) 引入了 {{cssxref(":target")}} [伪类](/zh-CN/CSS/Pseudo-classes). Netscape 7.1 已经在 Netscape 系列中加入了这个伪类的支持，这一新的举措让页面作者能够辅助用户在较大的页面中定位。
 
 ### 选择一个目标
 
-`{{ Cssxref(":target") }} 伪类用来指定那些包含片段标识符的 URI 的目标元素样式。` 例如，`http://developer.mozilla.org/en/docs/Using_the_:target_selector#Example` 这个 URI 包含了 `#Example 片段标识符。` 在 HTML 中，标识符是元素的 id 或者 name 属性，。由于这两者位于相同的命名空间，因此，这个示例 URI 指向的是文档顶层的 "Example" 。
+{{cssxref(":target")}} 伪类用来指定那些包含片段标识符的 URI 的目标元素样式。例如 `http://developer.mozilla.org/en/docs/Using_the_:target_selector#Example` 这个 URI 包含了 `#Example` 片段标识符 在 HTML 中，标识符是元素的 `id` 或者 `name` 属性，。由于这两者位于相同的命名空间，因此，这个示例 URI 指向的是文档顶层的 "Example" 。
```
Comment (yin1999) [inline files/zh-cn/web/css/css_selectors/using_the__colon_target_pseudo-class_in_selectors/index.md:13]: Let's also update the "broken" url here :)

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Using_the_:target_pseudo-class_in_selectors#picking_a_target

```suggestion
{{cssxref(":target")}} 伪类用来指定那些包含片段标识符的 URI 的目标元素样式。例如 `https://developer.mozilla.org/zh-CN/docs/Web/CSS#参考` 这个 URI 包含了 `#参考` 片段标识符 在 HTML 中，标识符是元素的 `id` 或者 `name` 属性。由于这两者位于相同的命名空间，因此，这个示例 URI 指向的是文档顶层的“参考”。
```

File: files/zh-cn/web/css/margin-top/index.md (hunk 1 above)
Comment (queengooborg) [reply files/zh-cn/web/css/margin-top/index.md]: I think that was a manual tweak I made, my bad!

**Review by yin1999 — APPROVED** [review]
I've allowed myself to format the markdown table. Thanks @queengooborg :)

**Pull request merged**